Source/TextKit/ASTextKitFontSizeAdjuster.mm

```diff
@@ -87,8 +87,6 @@ + (void)adjustFontSizeForAttributeString:(NSMutableAttributedString *)attrString
       paragraphStyle.tailIndent = (paragraphStyle.tailIndent * scaleFactor);
       paragraphStyle.minimumLineHeight = (paragraphStyle.minimumLineHeight * scaleFactor);
       paragraphStyle.maximumLineHeight = (paragraphStyle.maximumLineHeight * scaleFactor);
-      paragraphStyle.lineHeightMultiple = (paragraphStyle.lineHeightMultiple * scaleFactor);
-      paragraphStyle.paragraphSpacing = (paragraphStyle.paragraphSpacing * scaleFactor);
```

Comment:
`paragraphSpacing` is already being scaled in line 84. Repeating the scaling cases the paragraph spacing mismatch the font point scaling.